RadarSDK/Radar.m

```diff
@@ -41,7 +41,21 @@ + (id)sharedInstance {
     return sharedInstance;
 }
 
++ (void)migrateIfNeeded {
```

Comment:
Let's make this function name a bit more specific (in case we need future migrations)